cmd/observe/observe.go

```diff
@@ -37,9 +37,10 @@ var (
 	}
 
 	otherOpts struct {
-		ignoreStderr    bool
-		printRawFilters bool
-		inputFile       string
+		ignoreStderr          bool
+		printRawFilters       bool
+		inputFile             string
+		experimentalFieldMask string
```

Comment:
nit: making a new struct here for experimental opts is probably more clear in the long run